### PR TITLE
Refs #28067 - defaults for dynflow sidekiq redis

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: config/foreman.hiera/family/RedHat.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
 # Disable additional Phusion Passenger repo on EL, preferring EPEL/Foreman's
 apache::mod::passenger::manage_repo: false
+redis::params::default_install: false
```
Comment (ekohl) [inline config/foreman.hiera/family/RedHat.yaml]: I think we do want the default install. We count on that in the pulpcore module
```suggestion
```

File: config/foreman.hiera/family/RedHat.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+redis::params::manage_repo: false
```
Comment (ekohl) [inline config/foreman.hiera/family/RedHat.yaml]: This should be set on the redis class itself
```suggestion
redis::manage_repo: false
```

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by ehelms** [comment]
This PR and https://github.com/theforeman/foreman-installer/pull/434/files are attempting to do the same thing. However, I think based on comments #434 is more correct at this point.

**Comment by ezr-ondrej** [comment]
Closing in favor of #434

**Pull request closed without merging**